edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentController.cs

```diff
@@ -415,9 +404,9 @@ public static void UpdatePvc(V1PersistentVolumeClaim pvc)
                 Log.LogInformation((int)EventIds.UpdatePvc, $"Update PVC {pvc.Metadata.Name}");
             }
 
-            public static void PvcException(Exception ex)
+            public static void DeployModuleException(Exception ex)
```

Comment:
nit: Can we change accessibility of all exception logger methods to internal static void? It is reasonable code cleanup because we are introducing a potential conflicting name "DeployModule" here.